### DISCUSSION TITLE
Minimize button

### DISCUSSION THREAD

**Discussion by pokecoins19** [sonigy/YCS#51] [Ideas]
I would be nice if there was a minimize button so I could shrink the block down a bit.

**Comment by Letssharearow** [comment]
I thought this Idea was cool, So I wrote a script that adds a button to minimize the app

https://greasyfork.org/en/scripts/477860-ycs-minimize

**Comment by eedo31** [comment]
This is definitely a feature that should be implemented. It would be great to be able to minimize the menu when not in use. It takes up a good chunk of screen real estate, and I don't need to use it in every single video
